internal/wazeroir/compiler.go

```diff
@@ -263,11 +262,9 @@ operatorSwitch:
 		// Create a new frame -- entering loop.
 		frame := &controlFrame{
 			frameID:          c.nextID(),
-			originalStackLen: len(c.stack),
+			originalStackLen: len(c.stack) - len(bt.Params),
```

Comment:
@mathetake thanks tons for fixing all this and the added docs. This is the only one I'd like to doc better. Have a good answer for why we subtract the block type params from the original stack length?

Reply:
memory is really fuzzy lemme think about it a bit..

Reply:
renamed for clarity!